frontend/packages/ux-editor/src/containers/DesignView/AddItem/useSearchComponent/useSearchComponent.test.ts

```diff
@@ -0,0 +1,155 @@
+import { useSearchComponent } from './useSearchComponent';
+import { act } from 'react';
+import type { KeyValuePairs } from 'app-shared/types/KeyValuePairs';
+import type { IToolbarElement } from '../../../../types/global';
+import { ComponentType } from 'app-shared/types/ComponentType';
+import { renderHookWithProviders } from '../../../../testing/mocks';
+import { waitFor } from '@testing-library/react';
+import type { TestCase } from './types';
+
+const MockIcon = () => null;
+const textInputValue = 'Text';
+
+const mockAvailableComponents: KeyValuePairs<IToolbarElement[]> = {
+  category1: [
+    { type: ComponentType.TextArea, label: 'Text Component', icon: MockIcon },
+    { type: ComponentType.Button, label: 'Button Component', icon: MockIcon },
+    { type: ComponentType.Checkboxes, label: 'Avmerkingsbokser', icon: MockIcon },
```

Comment:
I can see that the values for `Text Component, `Button Component`, etc. is used multiple times in the file. What do you think about moving them into reusable variables? 😄